EnhancedMediaPlayer.podspec

```diff
@@ -0,0 +1,13 @@
+
+Pod::Spec.new do |s|
+    s.name             = 'EnhancedMediaPlayer'
+    s.version          = '0.1.0'
+    s.summary          = 'Simple and customizable media player for iOS'
+    s.homepage         = 'https://github.com/profusion/ios-enhanced-media-player/blob/main/EnhancedMediaPlayerSample/ContentView.swift'
+    s.license          = { }
```

Comment:
important and missing field, as this can cause us legal liability for this software. please publish this under `MIT` and assure for other distribution methods we also have this